crypto/evp/pmeth_lib.c

```diff
@@ -199,6 +199,7 @@ static EVP_PKEY_CTX *int_ctx_new(OSSL_LIB_CTX *libctx,
             }
 #ifndef FIPS_MODULE
             if (keytype != NULL) {
+                OPENSSL_init_crypto(OPENSSL_INIT_LOAD_CONFIG, NULL);
```

Comment:
Should we rather call `ossl_init_casecmp()` I am afraid of unwanted consequences of `OPENSSL_init_crypto(OPENSSL_INIT_LOAD_CONFIG, NULL);`

Reply:
In `EVP_PKEY_CTX_new_id` we call `OPENSSL_init_crypto` so we are sort of consistent here

Reply:
Hmm, right, the object functions trigger that call.